pyproject.toml

```diff
@@ -188,26 +188,25 @@ urllib3 = "^1.26.18"
 rq = "1.10.1"
 rules = "2.2"
 ujson = ">=3.0.0"
-xmljson = "0.2.0"
+xmljson = "0.2.1"
 colorama = ">=0.4.4"
 boxing = ">=0.1.4"
 redis = "~=3.5"
 sentry-sdk = ">=1.1.0"
 launchdarkly-server-sdk = "8.2.1"
 python-json-logger = "2.0.4"
-label-studio-converter = {url = "https://github.com/HumanSignal/label-studio-converter/archive/1abb2420cdbe9daf637b68b14162327662598dc2.zip"}
 google-cloud-storage = "^2.13.0"
 google-cloud-logging = "^3.10.0"
 mysqlclient = {version = "*", optional = true}
 django-csp = "3.7"
 openai = "^1.10.0"
+label-studio-sdk = "^1.0.0"
```

Comment:
label-studio-sdk = {url = "https://github.com/HumanSignal/label-studio-sdk/archive/36f4dfa4c0a9fbb6ee571469aa057ce221910e04.zip"}

Reply:
@farioas updated this with poetry add